api/templates/design.go

```diff
@@ -649,7 +649,10 @@ var VersionDetail = Type("VersionDetail", func() {
 		MinLength(1)
 		Example("Lorem markdownum quod discenda [aegide lapidem](http://www.nequeuntoffensa.io/)")
 	})
-	Required("components", "readme")
+	Attribute("components", ArrayOf(String), "List of components used in the template version.", func() {
+		Example([]string{"ex-generic-v2", "keboola.snowflake-transformation"})
+	})
+	Required("readme", "components")
```

Comment:
Vyhodil som som `components` s `Version` a nechal iba vo `VersionDetail`, ... tak ako je to v UI.
Potrebuju zoznam components `Template` -> templates index, ... a potom az na detaile verzie.

Inak v kode bola chyba a `Version.Components` sa nikde nenastavovalo, takze ziadna zmena v testoch.